packages/styles/panel.css

```diff
@@ -12,14 +12,26 @@
   background-color: var(--panel-background-color);
 }
 
-.Panel__Heading {
+.Panel__Heading,
+.Panel legend {
   font-size: var(--text-size-medium);
   color: var(--panel-heading-color);
   padding: 0;
   margin: 0 0 var(--space-large) 0;
   font-weight: var(--font-weight-bold);
 }
 
+fieldset.Panel {
+  position: relative;
+  padding-top: 76px;
+}
+
+/* reset absurd browser-default legend styles */
+.Panel legend {
```

Comment:
I've made it so the legend matches the exact styling of the Panel__Heading elements